fix: Add missing ring/bar interface properties (#64)
endless-sky/endless-sky-wiki: wiki/CreatingInterfaces.md

		from <x#> <y#> to <x#> <y#> [<anchor>]
		color <color>
		size <size#>
		[reversed]
		[start angle <angle#>]
		[span angle <angle#>>]
```
Defines a straight line (bar) or circular outline (ring) to be drawn with this interface.

> In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn.

Reviewer: ```suggestion
In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn **(v. 0.10.2)**.
```
https://github.com/endless-sky/endless-sky/commit/b0beb6e664d0de38d9a949521825201727114d09

> In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box. The start and span angle specifies the portion of the ring that is drawn **(v. 0.10.2)**.

Reviewer 2: New line?
Also, some more details would be good.
```suggestion
In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box.
Beginning in **v. 0.10.2**, it is also possible to define the "start angle" and "span angle" for a ring.
The "start angle" defines where along the radius filling of the ring should begin. The default value is 0, which corresponds to straight up. [Also mention which direction it goes in.]
[I'll need to go check the original PR to produce a better description of span angle. I'll fill these in later.]
```

Reviewer 2: @tibetiroka updated description above.

Author: > Values as low as 0 are allowed.

"The minimum value is 0, in which case nothing is drawn."

Reviewer 2: Done

A bar will be drawn from the bottom right corner of its bounding box.

> At runtime, the game may only partially complete the bar or ring, or segment it, for example, the ship hull status ring, or the fuel bar. If only a part is drawn, the reversed attribute can toggle which end it is filled from.

Reviewer: ```suggestion
At runtime, the game may only partially complete the bar or ring, or segment it, for example, the ship hull status ring, or the fuel bar. If only a part is drawn, the reversed attribute can toggle which end it is filled from **(v. 0.10.3)**.
```
https://github.com/endless-sky/endless-sky/commit/50b8aa70d65435dd3fd9b4a83069066f4e8dcd0b

> At runtime, the game may only partially complete the bar or ring, or segment it, for example, the ship hull status ring, or the fuel bar. If only a part is drawn, the reversed attribute can toggle which end it is filled from **(v. 0.10.3)**.

Reviewer 2: Same. I think it's more important here in order to indicate that the partial completion existed before this version.
```suggestion
At runtime, the game may only partially complete the bar or ring, or segment it, for example, the ship hull status ring, or the fuel bar.
Beginning in **v0.10.3**, "reversed" can be used to invert the fill direction. A reversed ring will be filled in the clockwise direction, and a reversed bar will be filled from the top left corner.
```

Author: We could also mention that a reversed horizontal bar is filled from the left side.

		from <x#> <y#> to <x#> <y#> [<anchor>]
		color <color>
		size <size#>
		[reversed]
		[start angle <angle#>]
		[span angle <angle#>>]
```
Defines a straight line (bar) or circular outline (ring) to be drawn with this interface.
In the case of a ring, it will be drawn anti-clockwise around the center of its bounding box, with a radius equal to half the width of the bounding box.
Beginning in **v. 0.10.2**, it is also possible to define the "start angle" and "span angle" for a ring.

> The "start angle" defines from how many degrees clockwise from a line straight up from the centre the ring should start filling. The default value is 0, which corresponds to straight up. Values of greater than or equal to zero and less than 360 are allowed.

Author: ```suggestion
The "start angle" defines from how many degrees clockwise from a line straight up from the center the ring should start filling. The default value is 0, which corresponds to straight up. Values of greater than or equal to zero and less than 360 are allowed.
```
